flatdata-go/backward-compatibility-tests/Gopkg.toml

```diff
@@ -0,0 +1,27 @@
+
+# Gopkg.toml example
```

Comment:
This lines can be removed?

Reply:
The file is autogenerated by `dep`. It can be removed, but if we will regenerate it again for some reason, then it will reappear.
I can remove it, for sure...